src/Assets/pageTitleTags.ts

```diff
@@ -1,32 +1,24 @@
-export type StepName =
-  | 'step-1'
-  | 'step-2'
-  | 'step-3'
-  | 'step-4'
-  | 'step-5'
-  | 'step-6'
-  | 'step-7'
-  | 'step-8'
-  | 'step-9'
-  | 'step-10'
-  | 'step-11'
-  | 'confirm-information'
-  | 'results';
+import { QuestionName } from '../Types/Questions';
 
-const pageTitleTags: Record<StepName, string> = {
-  'step-1': 'Preferred Language',
-  'step-2': 'Legal',
-  'step-3': 'Zip and County',
-  'step-4': 'Number of HH Members',
-  'step-5': 'Individual HH Member',
-  'step-6': 'Expenses',
-  'step-7': 'Assets',
-  'step-8': 'Existing Benefits',
-  'step-9': 'Near Term Help',
-  'step-10': 'Referral',
-  'step-11': 'Optional Sign Up',
-  'confirm-information': 'Confirmation',
-  results: 'Results',
+export const QUESTION_TITLES: Record<QuestionName, string> = {
+  zipcode: 'Zip and County',
+  householdSize: 'Number of HH Members',
+  householdData: 'Individual HH Member',
+  hasExpenses: 'Expenses',
+  householdAssets: 'Assets',
+  hasBenefits: 'Existing Benefits',
+  acuteHHConditions: 'Near Term Help',
+  referralSource: 'Referral',
+  signUpInfo: 'Optional Sign Up',
```

Comment:
What do you think about having Sydney create a ticket to move these to the back end config and translate them?

Reply:
Can you create an issue to discuss it. I don't think that we need to move them to the backend because they should be the same for each state, but I think that we would want to translate them. Although, I don't know how that would affect analytics or SEO. At the very least we should not abbreviate "household"